docs/secure-connections/sauce-connect-5/operation/kubernetes.md

```diff
@@ -7,46 +7,51 @@ sidebar_label: Kubernetes
 [Kubernetes](https://kubernetes.io/), an industry-standard environment for management of containerized applications, may be used for the automation of much of the operational effort required to run Sauce Connect Proxy.
 This article provides documentation for running Sauce Connect Proxy in [Kubernetes](https://kubernetes.io/).
 
-## Running Sauce Connect in Kubernetes
+## Helm Chart
 
 If you need a Sauce Connect Proxy to stay up indefinitely, we recommend using a [Helm chart](https://helm.sh/docs/topics/charts/) to manage your Sauce Connect Proxy instance or pool.
+The Sauce Connect 5 Helm chart is available at the [Sauce Labs Helm Charts Registry](https://opensource.saucelabs.com/helm-charts/SAUCE-CONNECT.html).
+The chart may be used as is, or adapted to your needs.
 
-The Sauce Connect Helm chart is available at the [Sauce Labs Helm Charts Registry](https://opensource.saucelabs.com/helm-charts/). The chart may be used as is, or adapted to your needs.
+### Usage
 
-### Using Sauce Connect Helm chart
+To use the chart, you must have a Kubernetes cluster running and [Helm](https://helm.sh/) installed.
 
-- Define a values file containing your configuration, for example:
+#### Define a required values file, for example:
 
 ```yaml
-sc:
-  region: "us-west"
-  user: johndoe
-  accessKey: "xxx-xxx-xxx"
-  tunnelName: "my-k8s-tunnel"
-tunnelPoolSize: 2
-terminationGracePeriodSeconds: 600
+config:
+  region: us-west
+  username: johndoe
+  access-key: "xxx-xxx-xxx"
+  tunnel-name: "my-k8s-tunnel"
```

Comment:
I'd add a note about `tunnelPoolSize` and `terminationGracePeriodSeconds` here. Those 2 are important and no default is good for all users.